test/suite/wtbound.py

```diff
@@ -130,9 +130,6 @@ def cursor_traversal_bound(self, cursor, lower_key, upper_key, next=None, expect
             if (self.lower_inclusive and lower_key):
                 self.assertTrue(self.check_key(lower_key) <= key)
             elif (lower_key):
-                # print("lower key:")
-                # print(lower_key)
-                # print(key)
```

Comment:
Did you intend to leave this change?

Reply:
Yes, we somehow never removed the debugging from that test file